src/TMxpClient.h

```diff
@@ -62,6 +62,9 @@ class TMxpClient
     virtual void setItalic(bool val) = 0;
     virtual void setUnderline(bool val) = 0;
 
+    virtual void setStyle(const QString& val) = 0;
```

Comment:
It isn't essential but I would like to see a comment at the top of these declarations pointing out to those not as familiar with all the nuances of C++ that these `= 0` values force this base class to be an abstract one (that cannot be instantiated itself) instead it means that ***any*** class derived from this one MUST implement the methods themselves to also avoid being *abstract*.

Reply:
If there's a wish too, I can easily add such a comment - even though I just copied from the setUnderline etc. above, which are not from me at all. Also, I've not the slightest idea in how many other different places in Mudlet this setup is used.
I've to admit though that I had to look up the exact meaning of this construction myself.